main.js

```diff
@@ -10,28 +29,17 @@ function fetchQuote () {
       withCredentials: false
     },
     success: function (response) {
-      console.log(response)
-      $('#quote').html(' " ' + response.quoteText)
-      tweetquote = response.quoteText
-      console.log(tweetquote)
-      if (response.quoteAuthor === '') {
-        $('#author').html(' - Unknown')
-      }
-      else {
-        $('#author').html(' - ' + response.quoteAuthor)
-      }
+      fetchSuccess(response)
     },
+
     error: function () {
-      alert('Could not retreive quote')
+      fetchError ()
     }
   })
 }
 
-var colors = ['#e67e22', '#3498db', '#9b59b6', '#e74c3c', '#2c3e50']
-
-var primColor = colors[Math.floor(Math.random() * colors.length)]
-
 function newColor () {
+  colors = ['#e67e22', '#3498db', '#9b59b6', '#e74c3c', '#2c3e50']
```

Comment:
move `var` declarations for colors and primColor in here so they never exist at the global scope